src/lib/Http/Session/Flash/NotificationBag.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Http\Session\Flash;
+
+use EzSystems\HybridPlatformUi\Notification\Notification;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Used for adding notifications.
+ */
+class NotificationBag
+{
+    const FLASH_MESSAGE_TYPE = 'hyybrid-notification';
+
+    /**
+     * @var FlashBagInterface
+     */
+    private $flashBag;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(Session $session, TranslatorInterface $translator)
+    {
+        $this->flashBag = $session->getFlashBag();
+        $this->translator = $translator;
+    }
+
+    /**
+     * Add success notification.
+     *
+     * @param string $message
+     * @param array $params
+     * @param null $domain
+     */
+    public function addSuccess(string $message, array $params = [], $domain = null)
```

Comment:
I think that we should have those methods available in the `TabController`, as abstracts. We may even want to _move_ the `Notification` object building to these abstracts. That way, we don't tie the type of notifications we support to this object, meaning that we can easily add more, including in 3rd party code.

Could even be part of a Trait that the abstract controller uses ?